doc/cephfs/cephfs-mirroring.rst

```diff
@@ -120,7 +120,9 @@ system, run a command of the following form:
 
 .. note:: "Mirroring module" commands are prefixed with ``fs snapshot mirror``.
    This distinguishes them from "monitor commands", which are prefixed with ``fs
-   mirror``. Be sure (in this context) to use module commands.
+   mirror``. Enabling mirroring through monitor commands would result in mirror daemon
```

Comment:
s/Enabling mirroring through monitor commands would result in mirror daemon "failed" state due to the absence of `cephfs_mirror` index object./Enabling mirroring by using monitor commands will result in the mirror daemon entering a "failed" state due to the absence of the ``cephfs_mirror`` index object./